bin/testEnvLocalK8S.sh

```diff
@@ -7,10 +7,15 @@ set -euo pipefail
 # Based on circleCI config - used to reproduce the environment and to improve local testing
 
 # expect istio scripts to be under $GOPATH/src/istio.io/istio/bin/...
-export TOP=$(cd $(dirname $0)/../../../..; pwd)
+#export TOP=$(cd $(dirname $0)/../../../..; pwd)
+export TOP=${GOPATH}
```

Comment:
GOPATH can have multiple components. 

But the correct version is ${TOP:-$(...)} - since TOP can be set beforehand when running, it is set by makefile.